exercises/D-predicates/exercise.js

```diff
@@ -6,10 +6,14 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return number <= 10;
```

Comment:
Hi Hozan, You did great work!
You can edit line No.15 to "return number >0 && number < 10;"  in order to check the number if is more than zero;

Reply:
As Somayeh said, you want to check the number is between 0 and 10 excluding the numbers 0 and 10.

Reply:
Hi Somayeh, Thanks for this suggestion, I will update the file :)